### DISCUSSION TITLE
Programm Partizipations-Meeting vom 1. Dezember 2023

### DISCUSSION THREAD

**Discussion by christianheimann** [e-id-admin/governance-sounding-board#50] [Announcements]
Zeit | Was
:-- | :--
ab 08.30 | Eintreffen zu Kaffee & Gipfeli
09.00 | Eröffnung durch Bundesrätin Elisabeth Baume-Schneider, EJPD und Dirk Lindenmann, BIT
09.30 | Gesetz & Botschaft; Michael Schöll, Urs Paul Holenstein, BJ
10.30 | Pause
11.00 | Projekte im Rahmen des E-ID-Programms; E-ID-Team
11.20 | "Das Potential von digitalen Nachweisen aus dem Privatsektor firmenübergreifend angewendet"; Désirée Heutschi, Orell Füssli AG, Dominic Schlegel, Swisscom AG
11.40 | "Die Sandbox lebt! Dank der SSImply Wallet und dem DVS TG4U Ökosystem"; Reto Schubnell, Kanton Thurgau
12.00 | "Google Wallet, an interoperable digital wallet for all"; Lucyna Janas, Google

**Mittagessen ab 12.45 Uhr**

Zeit | Was
:-- | :--
13.30 | World-Cafés (Procivis AG / Orell Füssli AG, Kanton Thurgau, HES-SO, esatus und BIT)
14.30 | Pause
15.00 | Diskussionpapier "Tech-Proposal", Christian Heimann, fedpol
15.20 | "Open Letter on the EU's proposed digital identity reform"; Annett Laube-Rosenpflanzer, BFH
15.30 | Update EU
15.45 | Daniel Säuberli, DIDAS
16.00 | Closing & Apéro

Für die Vor-Ort-Teilnahme werden keine Anmeldungen mehr entgegengenommen. 

**Wo?**
Physisch: Bundesamt für Informatik und Telekommunikation BIT, Eichenweg 3, 3052 Zollikofen
Online: Die Hauptpräsentationen werden online übertragen, Link folgt.

**WebEx-Link**
https://adminch.webex.com/adminch/j.php?MTID=ma3fc0e75c65a2420e95d8b02e5055843
___________

Heure | Sujet
:-- | :--
dès 08.30 | Arrivée pour café & croissants
09.00 | Ouverture par la Conseillère fédérale Elisabeth Baume-Schneider, DFJP, Dirk Lindenmann, OFIT
09.30 | Projet de loi & message; Michael Schöll, Urs Paul Holenstein, BJ
10.30 | Pause
11.00 | Projets dans le cadre du programme e-ID; Equipe e-ID
11.20 | "Le potentiel des preuves numériques du secteur privé appliqué à plusieurs entreprises"; Désirée Heutschi, Orell Füssli AG, Dominic Schlegel, Swisscom AG
11.40 | "Le Sandbox est vivant ! Grâce au portefeuille SSImply et à l'écosystème DVS TG4U"; Reto Schubnell, Canton de Thurgovie
12.00 | "Google Wallet, an interoperable digital wallet for all"; Lucyna Janas, Google

**Repas de midi dès 12h45**

 Heure | Sujet
:-- | :--
13.30 | World-Cafés (Procivis AG / Orell Füssli AG, Kanton Thurgau, HES-SO, esatus et OFIT)
14.30 | Pause
15.00 | Papier de discussion "Tech-Proposal"; Christian Heimann, fedpol
15.20 | "Open Letter on the EU's proposed digital identity reform"; Annett Laube-Rosenpflanzer, BFH
15.30 | Update EU
15.45 | Daniel Säuberli, DIDAS
16.00 | Closing & Apéro

Les inscriptions ne sont plus acceptées pour la participation sur place. 

**Où?**
Physique: Office fédéral de l'informatique et de la télécommunication OFIT, Eichenweg 3, 3052 Zollikofen
En ligne: Les présentations principales seront diffusées en ligne, lien à venir.

**WebEx-Link**
https://adminch.webex.com/adminch/j.php?MTID=ma3fc0e75c65a2420e95d8b02e5055843

**Comment by grt-dti** [comment]
Wo findet die Partizipation statt?

**Reply by christianheimann** [reply]
Das Partizipations-Meeting findet als hybride Veranstaltung statt.
Physisch: Bundesamt für Informatik und Telekommunikation BIT, Eichenweg 3, 3052 Zollikofen
Online: Die Hauptpräsentationen werden online übertragen, Link folgt.

**Reply by stephanekk** [reply]
Will there be an English or French translation?
Thanks

**Reply by christianheimann** [reply]
There is no simultaneous translation for the event. But the whole day will be multilanguage german, french, english. 
All slides from the E-ID team will be either german/french or english. The other speakers choose their language. (Google will do in english, EU very probably as well).
We will try to find the best balance between sharing the important things in german/french and avoiding unneccessary repetition in a second language... which is, honestly, not an easy task.
